src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -276,11 +277,24 @@ function trapBubbledEventsLocal() {
   var node = getNode(inst);
   invariant(node, 'trapBubbledEvent(...): Requires node to be rendered.');
 
+  function trapBubbledEvent(topLevelType, handlerBaseName, element) {
+    if (!element) {
+      return;
+    }
+    var callback = ReactDOMEventListener.dispatchEvent.bind(null, topLevelType);
+    element.addEventListener(handlerBaseName, callback, false);
+    return {
+      remove() {
+        element.removeEventListener(handlerBaseName, callback, false);
+      },
+    };
+  }
+
   switch (inst._tag) {
     case 'iframe':
     case 'object':
       inst._wrapperState.listeners = [
-        ReactBrowserEventEmitter.trapBubbledEvent('topLoad', 'load', node),
+        trapBubbledEvent('topLoad', 'load', node),
```

Comment:
Stack code still relies on return value for unsubscribing. Fiber doesn't. I inlined the old Stack-friendly version right into Stack.

Reply:
How does Fiber handle removing those listeners on unmount? Does it just not do that anymore?

Reply:
I think it doesn't. Relies on GC to clean them up.

Reply:
At some point I need to circle back about this on #9333. We need local listeners and I'm trying to figure out the best way to clean them up.